grammar.js

```diff
@@ -299,21 +301,28 @@ module.exports = grammar({
       ')'
     ),
 
-    lambda_parameters: $ => $._parameters,
+    lambda_parameters: $ => choice(
+      seq(
+        '(',
+        optional($._parameters),
+        ')'
+      ),
```

Comment:
I originally added the `empty_expression` rule to account for `lambda (): True`, and because `(lambda (tuple) (true))` is a bad tree. So in places where `()` can occur that are not tuples, I've updated the rules to match for parens.